Mathlib/SetTheory/Ordinal/Exponential.lean

```diff
@@ -406,8 +406,8 @@ theorem iSup_pow {o : Ordinal} (ho : 0 < o) : ⨆ n : ℕ, o ^ n = o ^ ω := by
   · exact (opow_isNormal ho₁).apply_omega
   · rw [one_opow]
     refine le_antisymm (Ordinal.iSup_le fun n => by rw [one_opow]) ?_
-    convert Ordinal.le_iSup _ 0
-    rw [Nat.cast_zero, opow_zero]
+    convert Ordinal.le_iSup (fun n ↦ (1 : Ordinal.{u_1}) ^ n) 0
+    exact opow_natCast 1 _
```

Comment:
Does this work/allow you to omit the function declaration?
```suggestion
    exact_mod_cast Ordinal.le_iSup _ 0
```
If not, is the universe annotation required? If it is, can you use a nicer universe like `u`?